uint16[] id_list
float64[] current_ma # max current
float64[] velocity_deg_s
float64[] profile_acc_deg_ss